cloudify_ansible/tasks.py

```diff
@@ -33,3 +38,16 @@ def run(playbook_args, _ctx, **_):
         _ctx.instance.runtime_properties['result'] = result.__dict__
     except CloudifyAnsibleSDKError:
         raise NonRecoverableError(CloudifyAnsibleSDKError)
+
+
+@operation
+@ansible_relationship_source
+def ansible_requires_host(new_sources_dict, _ctx, **_):
+    current_sources_dict = \
+        _ctx.source.instance.runtime_properties.get(
```

Comment:
@EarthmanT It would be an issue if we passed a node instance context instead of subject relation context

Reply:
What is the issue?

Reply:
It will be crashed if you passed NodeInstance Context instead of CLoudifyContext

Reply:
I think this one is not relevant, because this is only meant to be called in relationship.